### PR TITLE
[WNMGDS-2345] Reorganize story files so they're easier to find

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

WNMGDS-2345

Reorganize story files so they're easier to find, closer to the thing they're describing

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [x] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [x] Selected appropriate `Impacts`, multiple can be selected.

### CODE REVIEW

**Review by zarahzachz — APPROVED** [review]

File: .storybook/main.ts [hunk 1]
```diff
@@ -9,11 +9,7 @@ const config: StorybookConfig = {
     ${head}
     <link rel="stylesheet" type="text/css" title="themeCss" href="core-theme.css" />
   `,
-  stories: [
-    '../packages/design-system/src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
-    '../packages/ds-healthcare-gov/src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
-    '../packages/ds-medicare-gov/src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
-  ],
+  stories: ['../packages/*/(src|content)/**/*.stories.@(js|jsx|ts|tsx|mdx)'],
```
Comment (forrestbaer) [inline .storybook/main.ts:12]: Nice.

**Review by forrestbaer — APPROVED** [review]
Looks great, nice cleanup.

**Comment by pwolfert** [comment]
Looks like the `Alert` and `Badge` Storybook files are using the new syntax, and the story ids are getting renamed in the VRTs, but it's more than that; it also reorganizes them in the Storybook navigation:

Screenshot 2023-06-01 at 11 54 44 AM

**Comment by pwolfert** [comment]
Also when VRT refs were updated in 0341e7eefe9831fb3c6705502c0a8a8d9762a8aa, it showed that a bunch of _on dark_ stories for medicare were broken

**Comment by pwolfert** [comment]
@zarahzachz, @forrestbaer, and I were discussing our visual regression tests that have been flaky lately, specifically the "Reset Overview" story. Since [upgrading Playwright didn't work](https://github.com/CMSgov/design-system/pull/2514#issuecomment-1572906368), we're going to just remove that story ([54cf153](https://github.com/CMSgov/design-system/pull/2509/commits/54cf1535f027c0d4d43d55be5c70d1fb852ba9be))

**Pull request merged**